### PR TITLE
First handlebars templates MATHNET-26

### PR DESCRIPTION
**Pull request by Polej**

### CODE REVIEW

File: css/admin.css [hunk 1]
```diff
@@ -341,11 +341,23 @@ div.panel-accordion.show {
     position: absolute; top: 0; bottom: 0;
     right: 117px;
     border: 2px solid #999999; border-radius: 20px;
-    transition: all 0.3s ease-in 0s; 
+    transition: all 0.3s ease-in 0s;
 }
```
Comment (BarryOgg) [inline css/admin.css:376]: There should be empty lines between a closing bracket and the next selector. This applies not only to this one line, but to the whole file.

File: src/Admin.js [hunk 2]
```diff
@@ -1037,11 +1042,13 @@ class Admin {
...
                 this.views.$views_jsapp.empty();
 
-                let merge_view_update_toggle = '<div class="onoffswitch" style="display:none;"> <input type="checkbox" name="onoffswitch" '
-                    +'class="onoffswitch-checkbox" id="myonoffswitchmerge" onchange="liveUpdatesCheckboxChangeMerge(this);"> </input> <label class="onoffswitch-label" for="myonoffswitchmerge">'
-                    +'<span class="onoffswitch-inner"></span> <span class="onoffswitch-switch"></span> </label></div>';
+                let merge_view_update_toggle = this.templates.merge_view_update_toggle;
                 this.views.$views_jsapp.append(merge_view_update_toggle);
 
+                $('.onoffswitch-checkbox').on('change',(e)=>{
+                  this.liveUpdatesCheckboxChangeMerge(e,e.target);
```
Comment (BarryOgg) [inline src/Admin.js]: 1. Keep tabs consistent - if it's 4 spaces throughout the file, it shouldn't be 2 spaces here.
2. Please separate args with a space.

File: src/Admin.js [hunk 3]
```diff
@@ -1914,9 +1918,9 @@ class Admin {
     }
 
     //Called when the Live/Not Live Toggle is Set/Unset for the Merged View
-    liveUpdatesCheckboxChangeMerge(checkbox) {
+    liveUpdatesCheckboxChangeMerge(event,checkbox) {
```
Comment (BarryOgg) [inline src/Admin.js]: Ditto: put a space after the comma.

File: templates/handlebars/mergegroup.handlebars [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<div class="merge_group" style="visibility:hidden">
+    <h4> Merge Group</h4>
+    <div class="geogebrawebapplet group-merge-webapplet" id="appletContainer {{numgroups}}"></div>
```
Comment (BarryOgg) [inline templates/handlebars/mergegroup.handlebars]: There should be no spaces in element `id`s. Hyphenate it instead.

File: templates/handlebars/merge_view_update_toggle.handlebars [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<div class="onoffswitch" style="display:none;">
+    <input type="checkbox" name="onoffswitch" class="onoffswitch-checkbox" id="myonoffswitchmerge"> </input>
```
Comment (BarryOgg) [inline templates/handlebars/merge_view_update_toggle.handlebars]: Write `id`s in camelCase, preferably.

File: templates/handlebars/newgroup.handlebars [hunk 6]
```diff
@@ -0,0 +1,4 @@
+<div class="views_group_{{i}} col-md-4 col-sm-5 col-lg-4" >
```
Comment (BarryOgg) [inline templates/handlebars/newgroup.handlebars]: You need to write col-[size]-[num] only if you change the number on breakpoints, i.e. having both col-lg-4 aand col-md-4 is redundant, as the width doesn't change on md.

**Review by BarryOgg — CHANGES_REQUESTED** [review]

File: templates/handlebars/newgroup.handlebars (hunk 6 above)
Comment (Polej) [reply templates/handlebars/newgroup.handlebars]: So which one should be removed?

Comment (BarryOgg) [reply templates/handlebars/newgroup.handlebars]: IIRC, `col-lg-4`, make sure by resizing the window

Comment (Polej) [reply templates/handlebars/newgroup.handlebars]: Thanks

**Review by BarryOgg — APPROVED** [review]

**Pull request merged**